apis/infrastructure/v1beta1/byomachine_types.go

```diff
@@ -50,6 +56,10 @@ type ByoMachineStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
+	// HostInfo has the attached host platform details.
+	// +optional
+	HostInfo HostInfo `json:"hostinfo,omitempty"`
+
 	// +optional
 	Ready bool `json:"ready"`
```

Comment:
If the `Ready` status is optional should we also add `omitempty` or it will by default set it to `false`?

Reply:
I think we could add `omitempty`, but the zero value as `false` in this case seems reasonable to me. Do you have any thoughts?

Reply:
I think let's keep it as it is. 👍